src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -0,0 +1,67 @@
+package roomescape.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import roomescape.exception.InvalidReservationException;
+import roomescape.exception.NotFoundReservationException;
+import roomescape.model.Reservation;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicLong;
+
+
+@Controller
+public class ReservationController {
+
+    private final List<Reservation> reservations = new ArrayList<>();
+    private AtomicLong index = new AtomicLong(0);
+
+    @GetMapping("/reservation")
+    String reservation() {
+        return "reservation.html";
+    }
+
+    @ResponseBody
+    @GetMapping("/reservations")
+    public List<Reservation> read() {
+        return reservations;
+    }
+
+    @PostMapping("/reservations")
+    public ResponseEntity<Reservation> create(@RequestBody Reservation reservation) {
+        if (reservation.getName() == null || reservation.getDate() == null || reservation.getTime() == null) {
+            throw new InvalidReservationException();
+        }
+        Reservation newReservation = new Reservation(index.incrementAndGet(), reservation.getName(), reservation.getDate(), reservation.getTime());
+        reservations.add(newReservation);
+        return ResponseEntity.
+                created(URI.create("/reservations/" + newReservation.getId()))
+                .body(newReservation);
```

Comment:
별 거 아니긴 한데 한 줄에 점 하나씩 허용해서 아래처럼 많이 작성하는 것 같아요
```suggestion
        return ResponseEntity.created(URI.create("/reservations/" + newReservation.getId()))
                             .body(newReservation);
```

Reply:
아하, 코드 습관 교정 감사합니다!!